### PR TITLE
added tooltips to explain building shorthand under objectives tab

### LINKED ISSUE #1828: Add tooltips to objectives tab to explain abbreviations under 'Objective Damage'
So basically have the tooltip for `T1` be `Top Tier 1 Tower`, `M1` be `Middle Tier 1 Tower`, etc.

### PR DESCRIPTION
**Pull request by mobyte0**
fixes #1828

### CODE REVIEW

**Comment by mobyte0** [comment]
Response to issue #1828.

File: package-lock.json [hunk 1]
```diff
@@ -5986,8 +5986,7 @@
         },
```
Comment (howardchung) [inline package-lock.json]: probably should undo the changes to this file
